src/test/end-to-end/tests/performance/verified-login-subscription-test.ts

```diff
@@ -0,0 +1,16 @@
+Feature('Login - Performance');
+
+//This value can be changed to determine how many tests should run.
+//The number of concurrent logins can be changed via the 'run-workers' setting, in the package.json
+for (let i = 0; i < 10; i++) {
```

Comment:
Is this value 10 likely to be updated for performance run, or we mainly use the run-workers setting? If it needs to be changed frequently, are we better to use an environment variable?